delfin/drivers/driver_spec_schema.py

```diff
@@ -0,0 +1,519 @@
+                            'properties': {
+                                'unit': {'type': 'string', 'enum': ["MB/s"]},
+                                'description': {'type': 'string',
+                                                'minLength': 1,
+                                                'maxLength': 255}
+                            },
+                        },
+                        'responseTime': {
+                            'type': 'object',
+                            'properties': {
+                                'unit': {'type': 'string', 'enum': ["ms"]},
+                                'description': {'type': 'string',
+                                                'minLength': 1,
+                                                'maxLength': 255}
+                            },
+                        },
+                        'requests': {
+                            'type': 'object',
+                            'properties': {
+                                'unit': {'type': 'string', 'enum': ["IOPS"]},
+                                'description': {'type': 'string',
+                                                'minLength': 1,
+                                                'maxLength': 255}
+                            },
+                        },
+                        'serviceTime': {
+                            'type': 'object',
+                            'properties': {
+                                'unit': {'type': 'string', 'enum': ["ms"]},
+                                'description': {'type': 'string',
+                                                'minLength': 1,
+                                                'maxLength': 255}
+                            },
+                        },
+                        'readRequests': {
+                            'type': 'object',
+                            'properties': {
+                                'unit': {'type': 'string', 'enum': ["IOPS"]},
+                                'description': {'type': 'string',
+                                                'minLength': 1,
+                                                'maxLength': 255}
+                            },
+                        },
+                        'writeRequests': {
+                            'type': 'object',
+                            'properties': {
+                                'unit': {'type': 'string', 'enum': ["IOPS"]},
+                                'description': {'type': 'string',
+                                                'minLength': 1,
+                                                'maxLength': 255}
+                            },
+                        },
+                    },
+                    'additionalProperties': False
+                },
+            },
+            'additionalProperties': False
+        },
+    },
+    'additionalProperties': False,
+    'required': ['name', 'vendor', 'model']
```

Comment:
Can we validate including all required fields (https://json-schema.org/understanding-json-schema/reference/object.html#required-properties)

Reply:
The check is to mandate adding name, vendor and model filed, all other filed are optional.
@joseph-v I am not very clear on the comment. Please let me know if above scenario make sense. 
Would like to get more input on validation :)